dev/conductor/core/lib/src/globals.dart

```diff
@@ -127,12 +127,7 @@ List<String> getValuesFromEnvOrArgs(
     return env[envName]!.split(',');
   }
   final List<String> argValues = argResults[name] as List<String>;
-  if (argValues != null) {
```

Comment:
Actually, I think the bug here is that the cast on 129 should be nullable:

```dart
  final List<String> argValues = argResults[name] as List<String>?;
```

Reply:
Oh, totally. Fixed. Thanks for the catch!